libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.html

```diff
@@ -11,115 +11,107 @@
     }"
   >
     <sky-input-box
-      [disabled]="disabled"
+      [hasErrors]="hasErrors"
       [helpPopoverContent]="helpPopoverContent"
       [helpPopoverTitle]="helpPopoverTitle"
+      [hintText]="hintText"
       [labelText]="
         label || ('skyux_date_range_picker_default_label' | skyLibResources)
       "
-      [hintText]="hintText"
     >
       <select
-        class="sky-form-control"
         formControlName="calculatorId"
-        [attr.id]="dateRangePickerId + '-select-calculator'"
-        (blur)="onFieldBlur()"
+        [required]="isRequired()"
+        (blur)="onBlur()"
       >
         <option
           *ngFor="let calculator of calculators"
           [value]="calculator.calculatorId"
         >
-          {{ calculator.shortDescription }}
+          {{
+            calculator._shortDescriptionResourceKey
+              ? (calculator._shortDescriptionResourceKey | skyLibResources)
+              : calculator.shortDescription
+          }}
         </option>
       </select>
     </sky-input-box>
   </div>
   <div
-    class="sky-date-range-picker-form-group"
+    class="sky-date-range-picker-form-group sky-date-range-datepicker-wrapper"
+    [hidden]="!showStartDatePicker"
     [ngClass]="{
       'sky-date-range-picker-last-input':
         showStartDatePicker && !showEndDatePicker
     }"
-    [hidden]="!showStartDatePicker"
   >
-    <sky-input-box [disabled]="disabled">
-      <label
-        class="sky-control-label"
-        [attr.for]="dateRangePickerId + '-start-date'"
-        [ngClass]="{ 'sky-control-label-required': startDateRequired }"
-      >
-        {{
-          selectedCalculator?.type
-            | skyDateRangePickerStartDateResourceKey
-            | skyLibResources
-        }}
-      </label>
-
+    <sky-input-box
+      [hasErrors]="hasErrors"
+      [labelText]="
```

Comment:
Using input box "easy mode" so that most of the one-off features of the template can be removed.